src/main/java/org/apache/maven/plugins/ejb/IncludesExcludes.java

```diff
@@ -46,33 +46,27 @@ public IncludesExcludes( List<String> includes, List<String> excludes, List<Stri
 
     public String[] resultingIncludes()
     {
-        String[] result = new String[0];
-        if ( includes.isEmpty() )
-        {
-            result = defaultIncludes.toArray( new String[defaultIncludes.size()] );
-        }
-        else
-        {
-            result = includes.toArray( new String[includes.size()] );
-        }
-
-        return result;
+        return resultingXcludes( includes, defaultIncludes );
     }
 
     public String[] resultingExcludes()
+    {
+        return resultingXcludes( excludes, defaultExcludes );
+    }
+
+    private static String[] resultingXcludes( List<String> currentXcludes, List<String> defaultXcludes )
```

Comment:
can the args have a decent names?

Reply:
> can the args have a decent names?

Could you suggest something else that will cover both includes and excludes?

Reply:
Forget my comment, it was a shortsight. I will rereview.